models/data_entities.go

```diff
@@ -71,22 +70,23 @@ type PractitionerResourceDao struct {
 
 // PractitionerResourceDto contains the data for the practitioner resource in Mongo
 type PractitionerResourceDto struct {
+	ID   string                  `bson:"practitioner_id"`
 	Data PractitionerResourceDao `bson:"data"`
 }
 
 // AppointmentResourceDto contains the data for the appointment resource in Mongo
 type AppointmentResourceDto struct {
+	ID   string                 `bson:"practitioner_id"`
```

Comment:
Could the struct field name also be called PractitionerId for clarity please?

Reply:
BSON already given the definition of what the ID represent, however changed as requested